packages/global-cli/package.json

```diff
@@ -25,5 +21,11 @@
   },
   "jest": {
     "displayName": "global-cli"
+  },
+  "homepage": "https://github.com/react-native-community/cli/tree/master/packages/global-cli",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/react-native-community/cli.git",
+    "directory": "packages/global-cli"
```

Comment:
Can you please revert this one? We don't touch the global-cli package and this would make lerna pick up a new version and try to release it

Reply:
Reverted in [`2dc264e` (#1381)](https://github.com/react-native-community/cli/pull/1381/commits/2dc264e4815d4147501e424b2c9c332de52d10a7)